colorama is used only for os.name=='nt', so don't require it on other OS
(and don't require strict six version)
--- rednose-1.3.0/setup.py.orig	2018-02-11 19:07:53.000000000 +0100
+++ rednose-1.3.0/setup.py	2020-06-06 16:25:56.261712710 +0200
@@ -12,8 +12,8 @@
     packages=find_packages(exclude=['test', 'test.*']),
     description='coloured output for nosetests',
     entry_points={'nose.plugins.0.10': ['NOSETESTS_PLUGINS = rednose:RedNose']},
-    install_requires=['setuptools', 'termstyle >=0.1.7', 'colorama'],
-    tests_require=['six==1.10.0'],
+    install_requires=['setuptools', 'termstyle >=0.1.7', 'colorama;platform_system=="Windows"'],
+    tests_require=['six>=1.10.0'],
     long_description=read('README.rst'),
     name='rednose',
     py_modules=['rednose'],
